cime_config/testdefs/testmods_dirs/clm/matrixcnOn/include_user_mods

```diff
@@ -1 +0,0 @@
-../default
```

Comment:
Remove the file entirely?

Reply:
It is deleted; the Github view is just a bit confusing.

Reply:
Yes, I should have also noticed "This file was deleted" before clicking to see the diffs...